cypresse2e/cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+// ***********************************************************
```

Comment:
Another example file? should we remove or will you add stuff here?

Reply:
Hi @idjevm 
This file will be used to put plugin modules configuration like:
module.exports = (on, config) => {
  /** the rest of your plugins... **/
  require('cypress-log-to-output').install(on)
}
We will add stuff here.